lib/open_weather/models/one_call/alert.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module OpenWeather
+  module Models
+    module OneCall
+      class Alert < Model
+        property 'sender_name'
+        property 'event'
+        property 'start', transform_with: ->(v) { Time.at(v).utc } # UTC
+        property 'end', transform_with: ->(v) { Time.at(v).utc } # UTC
+        property 'description'
+
+        def initialize(args = nil, options = {})
```

Comment:
You don't need to override the constructor unless you're doing something extra here. Just remove it.